sdl_viewer/src/main.rs

```diff
@@ -211,21 +211,27 @@ impl NodeView {
 // Keeps track of the nodes that were requested in-order and loads then one by one on request.
 struct NodeViewContainer {
     node_views: HashMap<octree::NodeId, NodeView>,
-    requested: HashSet<octree::NodeId>,
+    requested: HashSet<octree::NodeId>,     // The node_ids that the I/O thread is currently loading.
```

Comment:
move comment before member. Maybe we should fix the column width we want to retain...